sds/source/sdsio/serial/cmsis_usart/sdsio_serial.c

```diff
@@ -0,0 +1,387 @@
+/*
+ * Copyright (c) 2023 Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// SDS I/O interface via Serial Port (CMSIS Driver:USART)
+
+#include <string.h>
+
+#include "cmsis_os2.h"
+
+#include "sdsio.h"
+#include "Driver_USART.h"
+
+// USART Driver Number
+#ifndef SDSIO_USART_DRIVER_NUMBER
+#define SDSIO_USART_DRIVER_NUMBER 1
```

Comment:
Should we use 0 as the default driver number?

Reply:
Yes, 0 is better for default value.